src/java.base/share/classes/java/lang/FdLibm.java

```diff
@@ -779,7 +780,6 @@ public static double compute(double x) {
      * shown.
      */
     static class Log10 {
-        private static double two54     = 0x1.0p54;              // 1.80143985094819840000e+16;
         private static double ivln10    = 0x1.bcb7b1526e50ep-2;  // 4.34294481903251816668e-01
 
         private static double log10_2hi = 0x1.34413509f6p-2;     // 3.01029995663611771306e-01;
```

Comment:
Are this constants intentionally not `final` ?

Reply:
No, just an oversight; I'll correct that. Thanks.